presto-main/src/test/java/com/facebook/presto/operator/TestHashAggregationOperator.java

```diff
@@ -453,10 +455,9 @@ public void testHashBuilderResizeLimit(boolean hashEnabled)
     @Test(dataProvider = "hashEnabled")
     public void testMultiSliceAggregationOutput(boolean hashEnabled)
     {
-        // estimate the number of entries required to create 1.5 pages of results
+        // estimate the number of entries required to create 1.3 pages of results
         int fixedWidthSize = SIZE_OF_LONG + SIZE_OF_DOUBLE + SIZE_OF_DOUBLE;
-        int multiSlicePositionCount = (int) (1.5 * PageBuilderStatus.DEFAULT_MAX_PAGE_SIZE_IN_BYTES / fixedWidthSize);
-        multiSlicePositionCount = Math.min((int) (1.5 * DEFAULT_MAX_BLOCK_SIZE_IN_BYTES / SIZE_OF_DOUBLE), multiSlicePositionCount);
+        int multiSlicePositionCount = (int) (1.3 * DEFAULT_MAX_PAGE_SIZE_IN_BYTES / fixedWidthSize);
```

Comment:
I don't think this is proper fix.

Reply:
@haozhun : Fixed now, see the first commit :)